lib/audit.js

```diff
@@ -189,8 +189,8 @@ function auditCmd (args, cb) {
   }).then((auditReport) => {
     return audit.submitForFullReport(auditReport)
   }).catch((err) => {
-    if (err.statusCode === 404 || err.statusCode >= 500) {
```

Comment:
401 needs to not do this, as the main registry is (or is about to) start returning 401s when your auth data is bad, regardless of endpoint.

Reply:
I would suggest that at the very least, we catch 404, 401 and then all other errors separately and tune the error messages to align better, eg:

401 - "Either your login credentials are invalid or your registry does not support audit."
404 - "Your registry does not support audit."
all other - what you have now

Reply:
I've changed this to follow your suggestion. You can re-review at your convenience.